container/layouts.go

```diff
@@ -102,6 +102,7 @@ func NewPadded(objects ...fyne.CanvasObject) *fyne.Container {
 
 // NewStack returns a new container that stacks objects on top of each other.
 // Objects at the end of the container will be stacked on top of objects before.
+// A stack with a single object does not change the size or position of that object.
```

Comment:
This doesn't seem right - it will resize it after all. It's more accurate to say that it doesn't provide any advantage as that is the default behaviour?

Reply:
Yeah, you're right. I meant that it wouldn't change the size. I didn't know how to word that in a good way. Will have to think a bit more about it.

Reply:
Why not just "A stack with a single object is a no-op" similar to the message you added to the layout itself?

Reply:
I wanted something more descriptive for the more commonly used container but maybe that was just a simple case of overthinking :)

Reply:
Maybe along the lines of "having only a single item has no impact as widgets will fill available space without Stack"?

Reply:
Sounds perfect. Thanks :)